### PR TITLE
midi for light (controller)

### PR DESCRIPTION
**Pull request by dg3nec**
new PR for mixxx 2.0, (was PR befor in Master)

### CODE REVIEW

File: res/controllers/Midi_for_light.midi.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	<info>
+		<name>MIDI for light</name>
+		<author>Michael Stahl / DG3NEC</author>
+		<description>MIDI-output for light control</description>
+        <forums>http://mixxx.org/forums/viewtopic.php?f=7&amp;t=7746</forums>
+        <wiki>http://mixxx.org/wiki/doku.php/midi_for_light</wiki>
+	</info>
+	<controller id="midi_for_light Control">
+		<scriptfiles>
+			<file filename="midi_for_light-scripts.js" functionprefix="midi_for_light"/>
```
Comment (rryan) [inline res/controllers/Midi_for_light.midi.xml]: Capitalization is off for the filename: ```Midi_for_light-scripts.js```

File: res/controllers/Midi_for_light.midi.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MixxxMIDIPreset mixxxVersion="1.12+" schemaVersion="1">
```
Comment (Pegasus-RPG) [inline res/controllers/Midi_for_light.midi.xml]: Change the Mixxx version to 2.0.0 :)

**Comment by Pegasus-RPG** [comment]
For reference, there is code within Mixxx that determines the current deck, but there's currently no way for scripts to access that information. (Bug https://bugs.launchpad.net/mixxx/+bug/1528644) So in the future you will be able to remove that logic from this script.
Otherwise, this LGTM.

**Comment by Be-ing** [comment]
I don't really know enough about MIDI clocks to review this, but I'll go through the wiki page to clarify the language. I may not get to that until after the 2.0 release.

**Comment by rryan** [comment]
Thanks @dg3nec !

**Comment by HorstBaerbel** [comment]
So, Uhm. I got [this](https://github.com/HorstBaerbel/NerDisco) which a simple OpenGL VJ solution. How would I connect it to Mixxx to get audio input? I don't quite get how to interface it and the wiki doesn't help... Any hints?

**Pull request merged**